site/docs-md/cordova/known-incompatible-plugins.md

```diff
@@ -0,0 +1,25 @@
+# Known Incompatible Cordova Plugins
+
+While we've tested a number of popular Cordova plugins, it's possible Capacitor doesn't support every Cordova plugin. Some don't work with Capacitor or Capacitor provides a conflicting alternative. If it's known that the plugin is conflicting or causes build issues, it will be skipped when running `npx cap update`.
+
+If you find an issue with an existing Cordova plugin, please [let us know](https://github.com/ionic-team/capacitor/issues/new) by providing the issue's details and plugin information.
+
+## Known incompatible plugins (Subject to change)
+
+- cordova-plugin-add-swift-support
+- cordova-plugin-admobpro
+- cordova-plugin-background-fetch
+- cordova-plugin-braintree
+- cordova-plugin-compat
+- cordova-plugin-console
+- cordova-plugin-crosswalk-webview
+- cordova-plugin-fcm
+- cordova-plugin-firebase
+- cordova-plugin-ionic
```

Comment:
@jcesarmobile - is this correct? this is Ionic Appflow/Deploy

Reply:
no, `cordova-plugin-ionic` is not incompatible